org-roam-capture.el

```diff
@@ -281,7 +281,7 @@ This function is used solely in Org-roam's capture templates: see
 (defun org-roam-capture--convert-template (template)
   "Convert TEMPLATE from Org-roam syntax to `org-capture-templates' syntax."
   (pcase template
-    (`(,key ,description) template)
+    (`(,_key ,_description) template)
     (`(,key ,description ,type ,target . ,rest)
```

Comment:
You beat me to it, but shouldn't we do the same thing for the next line?

Reply:
Apparently not.  I'm dumb, it's because they're clearly unused, here.

Reply:
Couldn't we just use `_`?

Reply:
They're used in `converted`, so byte compiler should not warn.

Reply:
> Couldn't we just use `_`?

Trying it out in a6029564e041495ef432d8e326866781d366e7a2.

Reply:
🎉

Reply:
We could, prefixing them with the `_` just signals to the byte-compiler that they're ignored variables. The names just help the reader understand the intent.

Reply:
Sure thing, I'll revert.